src/install/pre_install.sh

```diff
@@ -10,6 +10,10 @@ elif [ "${CODENAME}" = "sarah" ] || [ "${CODENAME}" = "serena" ] || [ "${CODENAM
 elif [ "${CODENAME}" = "rebecca" ] || [ "${CODENAME}" = "rafaela" ] || [ "${CODENAME}" = "rosa" ]; then
     CODENAME=trusty
     sudo apt-get -y install "linux-image-extra-$(uname -r)" linux-image-extra-virtual
+elif  [ "${CODENAME}" = "kali-rolling" ] || [ "${CODENAME}" = "buster" ]; then
```

Comment:
overwriting CODENAME "buster" with "buster" does not break anything. However you do not need to do it ;)